File: Codechef/Enormous Input Test/Enormous Input Test.cpp
#include <iostream> 

using namespace std;

int main() {
	// Read the input n, k
	int n, k;
	scanf("%d %d", &n, &k);

	// ans denotes number of integers n divisible by k
	int ans = 0;

	for (int i = 0; i < n; i++) {
		int t;
		cin>>t;

		if (t % k == 0) {
			ans++;
		}		
	}

	// Print the ans.
	cout<<ans<<endl;

	return 0;
}

File: Codechef/Enormous Input Test/Enormous Input Test.py
#Reading input n,k
n,k = map(int, input().split())
count = 0    #count denotes number of integers divisible by 3

for _ in range(0, n):
    t = int(input())
    #check for divisibility by 3
    if t%k==0:
    #If t is divisible by 3 increment count by 1
        count+=1
        
#print answer
print(count)
